geolocation-API/clearWatch_TypeError.https.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<title>Geolocation Test: clearWatch TypeError tests</title>
+<link rel="help" href="http://www.w3.org/TR/geolocation-API/#clear-watch" />
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+  test(() => {
+    for (const invalidId of [NaN, -1, 0, 1, 2147483648, Infinity, -Infinity]) {
+      navigator.geolocation.clearWatch(invalidId);
```

Comment:
Slight nit: This test would generate nicer failure notices if it created one test case for each invalid value it tried to pass to `clearWatch()`.

Reply:
Agree... I'll move the for outside the test.